cli/azd/cmd/monitor.go

```diff
@@ -58,29 +59,32 @@ func newMonitorCmd() *cobra.Command {
 }
 
 type monitorAction struct {
-	azdCtx      *azdcontext.AzdContext
-	env         *environment.Environment
-	subResolver account.SubscriptionTenantResolver
-	azCli       azcli.AzCli
-	console     input.Console
-	flags       *monitorFlags
+	azdCtx                      *azdcontext.AzdContext
+	env                         *environment.Environment
+	subResolver                 account.SubscriptionTenantResolver
+	azCli                       azcli.AzCli
+	deploymentOperationsService azapi.DeploymentOperations
```

Comment:
Should we just call this `deploymentOperations` and elide the "service" suffix here for the field?